p/scripts/main.js

```diff
@@ -1312,12 +1318,10 @@ function init_simple_scrollbar() {
 
 var scrollTimeout;
 function init_sticky_sidebar(){
```

Comment:
I'm away from my dev environment until Monday, so I'm not in a position to test this right now, but I think we still need to be doing the scroll function with native scrollbars, to ensure that the bottom of the scroll doesn't extend under the nav_entries, if they're visible. It's a minor thing, since with the padding at the end of the tree, no content should be obscured by the buttons; the bottom of the scrollbar might just slide under them.

Reply:
Indeed, the scrollbar slides under the buttons, and that seems to work well. The length of the scrollbar is slightly wrong in this case, but that is a minor issue which can likely be fixed purely in CSS if wanted (with a `calc()`) and later even better if/when moving to e.g. Grid + Flexbox.